.github/workflows/ci.yml

```
@@ -2,13 +2,8 @@ name: CI
 
 on:
   push:
-    branches-ignore:
-      - "github-pages/*"
-      - "gh-pages/*"
-      - "main"
-      - "master"
-  pull_request:
-    types: [opened, reopened]
+    branches:
+      main
```

```suggestion
    branches-ignore:
      - "github-pages/*"
      - "gh-pages/*"
      - "main"
      - "master"
```

see here https://github.com/exasol/python-toolbox/blob/7a08471810e66b16dc5cf54c4911f284f4f9ee6e/exasol/toolbox/templates/github/workflows/ci.yml#L3